Integrations/integration-UrlScan.yml

```diff
@@ -205,7 +205,6 @@ script:
             cont['Certificates'] = cert_ec
         url_cont['Data'] = url_query
         if 'urls' in scan_lists:
-            cont['RelatedURLs'] = scan_lists['urls']
```

Comment:
why are we removing this?

Reply:
Should be fixed now. With regards to the context outputs, they were vague so I changed the description to match what they actually do as well as removed the outputs that no longer exist.